docs/imagecustomizer/quick-start/container.md

```diff
@@ -43,7 +55,7 @@ docker run \
     --output-image-file "/mnt/staging/out/image.vhdx"
 ```
 
-Docker options:
+**Docker options:**
```

Comment:
I think I prefer these list headings without bold font. It avoids them looking like headers, which visually separates them from the example.